app/containers/App/App.jsx

```diff
@@ -280,95 +331,17 @@ class App extends React.Component {
-            component={DocumentEditView}
-            config={state.app.config}
-            mustBeSignedIn
-            path={`/:collection${REGEX_SLUG}/new/:section?`}
-          />
-
-          <Route
-            isSignedIn={isSignedIn}
-            exact
-            component={DocumentEditView}
-            config={state.app.config}
-            mustBeSignedIn
-            path={`/:group${REGEX_SLUG}/:collection${REGEX_SLUG}/new/:section?`}
-          />
-
-          <Route
-            isSignedIn={isSignedIn}
-            exact
-            component={DocumentEditView}
-            config={state.app.config}
-            mustBeSignedIn
-            path={`/:collection${REGEX_SLUG}/:documentId${REGEX_DOCUMENT_ID}/:section?`}
-          />
-
-          <Route
-            isSignedIn={isSignedIn}
-            exact
-            component={DocumentEditView}
-            config={state.app.config}
-            mustBeSignedIn
-            path={`/:group${REGEX_SLUG}/:collection${REGEX_SLUG}/:documentId${REGEX_DOCUMENT_ID}/:section?`}
-          />
-
-          <Route
-            isSignedIn={isSignedIn}
-            exact
-            component={DocumentListView}
-            config={state.app.config}
-            mustBeSignedIn
-            path={`/:collection${REGEX_SLUG}/:page${REGEX_NUMBER}?`}
-          />
-
-          <Route
-            isSignedIn={isSignedIn}
-            exact
-            component={ReferenceSelectView}
-            config={state.app.config}
-            mustBeSignedIn
-            path={`/:collection${REGEX_SLUG}/new/select/:referenceField/:page${REGEX_NUMBER}?`}
-          />
-
-          <Route
-            isSignedIn={isSignedIn}
-            exact
-            component={ReferenceSelectView}
-            config={state.app.config}
-            mustBeSignedIn
-            path={`/:collection${REGEX_SLUG}/:documentId${REGEX_DOCUMENT_ID}/select/:referenceField/:page${REGEX_NUMBER}?`}
-          />
-
-          <Route
-            isSignedIn={isSignedIn}
-            exact
-            component={DocumentListView}
-            config={state.app.config}
-            mustBeSignedIn
-            path={`/:group${REGEX_SLUG}/:collection${REGEX_SLUG}/:page${REGEX_NUMBER}?`}
-          />
-
-          <Route
-            isSignedIn={isSignedIn}
-            exact
-            component={ReferenceSelectView}
-            config={state.app.config}
-            mustBeSignedIn
-            path={`/:group${REGEX_SLUG}/:collection${REGEX_SLUG}/new/select/:referenceField/:page${REGEX_NUMBER}?`}
-          />
-
-          <Route
-            isSignedIn={isSignedIn}
-            exact
-            component={ReferenceSelectView}
-            config={state.app.config}
-            mustBeSignedIn
-            path={`/:group${REGEX_SLUG}/:collection${REGEX_SLUG}/:documentId${REGEX_DOCUMENT_ID}/select/:referenceField/:page${REGEX_NUMBER}?`}
-          />
+          {collectionRoutes.map(({path, component}) => (
+            <Route
```

Comment:
Do you mind sorting the props alphabetically here, please? (I know they weren't before.)